### PR TITLE
Positioning for Limitations section in pgd 4 and 5

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Limitations sections in Planning for PGD 4 and 5 now exist with placeholder text in the sections.

This is a for discussion PR.

### CODE REVIEW

File: product_docs/docs/pgd/4/limitations.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+-   Replacing a node with its physical standby doesn't work for nodes that use CAMO/Eager/Group Commit. Combining physical standbys and BDR in general isn't recommended, even if otherwise possible.
+
+-   A `galloc` sequence might skip some chunks if the sequence is created in a rolled back transaction and then created again with the same name. This can also occur if it is created and dropped when DDL replication isn't active and then it is created again when DDL replication is active. The impact of the problem is mild, because the sequence guarantees aren't violated. The sequence skips only some initial chunks. Also, as a workaround you can specify the starting value for the sequence as an argument to the `bdr.alter_sequence_set_kind()` function.
+
+-   Legacy BDR synchronous replication uses a mechanism for transaction confirmation different from the one used by CAMO, Eager, and Group Commit. The two are not compatible and must not be used together. Therefore, nodes that appear in `synchronous_standby_names` must not be part of CAMO, Eager, or Group Commit configuration. Using synchronous replication to other nodes, including both logical and physical standby is possible.
```
Comment (PJMODOS) [inline product_docs/docs/pgd/4/limitations.mdx]: Could we get rid of the "Using synchronous replication to other nodes, including both logical and physical standby is possible.", we say in the first paragraph that we don't recommend combining with physical replication.

Comment (PJMODOS) [reply product_docs/docs/pgd/4/limitations.mdx]: And yes that problem exists with current docs, but if we are fixing things...

Comment (djw-m) [reply product_docs/docs/pgd/4/limitations.mdx]: The sentence is now gone.

File: product_docs/docs/pgd/4/limitations.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+title: "Limitations"
+---
+
+
+This section covers design limitations of BDR, that should be taken into account
+when planning your deployment.
+
+## Limits
+
+- BDR can run hundreds of nodes on good-enough hardware and network. However, for mesh-based deployments, we generally don't recommend running more than 32 nodes in one cluster. Each master node can be protected by multiple physical or logical standby nodes. There's no specific limit on the number of standby nodes, but typical usage is to have 2&ndash;3 standbys per master. Standby nodes don't add connections to the mesh network, so they aren't included in the
```
Comment (PJMODOS) [inline product_docs/docs/pgd/4/limitations.mdx]: This sentence is heavily outdated, a) we don't recommend physical standbys at all, b) we generally don't recommend logical standbys other than when user really wants to save on cross-region bandwidth, and typical usage is to not use any standbys or any other special kind of node and just have data nodes.

Comment (djw-m) [reply product_docs/docs/pgd/4/limitations.mdx]: Suggesting:

We recommend that generally users only run data nodes. Standby node configurations are possible, but are only appropriate for situations where inter-regional bandwidth costs are a concern.  

or:

We recommend that generally users only run data nodes. Use of standby nodes is supported but not recommended.

Comment (djw-m) [reply product_docs/docs/pgd/4/limitations.mdx]: New language applied.

**Pull request merged**